src/main/webapp/student/play-game.html

```diff
@@ -54,6 +54,10 @@
   <link rel="stylesheet" href="../styles/main.css">
   <link rel="stylesheet" href="styles/play-game.css">
 
+  <script
+  src="https://code.jquery.com/jquery-3.5.1.js"
+  integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc="
+  crossorigin="anonymous"></script>
```

Comment:
Why did you add JQuery?

Reply:
Oh I was going to use it but let me delete it, thanks!